### ISSUE TITLE
Fee transparency is lacking

### ISSUE THREAD

**Issue by corneliusroemer** [liberapay/liberapay.com#1935]
I very much like the idea of Liberapay.

However: I do not like that fees are totally intransparent for donors. You just indicate "High fees/moderate fees" but I want concrete numbers. If you can't give exact %, at least give a numeric estimate. Otherwise I can't make a conscious choice between payment methods (bank transfer vs card vs paypal and small/many vs large/few vs none at all).

**Comment by techtonik** [comment]
Calculation of fees requires some kind of loop between receiving account and Liberay to compare the value sent with the value received.

**Comment by corneliusroemer** [comment]
Yes and no, I wouldn't make it too complicated. Rough figures would be enough for me.

**Comment by techtonik** [comment]
You can create a static page and maintain it manually I guess. Figures can change if not traced automatically.
